src/imars3dv2/filters/denoise.py

```diff
@@ -0,0 +1,208 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+import numpy as np
+import tomopy
+import tomopy.util.mproc as mproc
+import concurrent.futures as cf
+from scipy.signal import convolve2d
+from scipy.ndimage import median_filter
+from skimage.restoration import denoise_bilateral
+
+
+def measure_noiseness(image: np.ndarray) -> float:
+    """
+    Measure the noiseness of the image, doi:10.1006/cviu.1996.0060
```

Comment:
Please format the doi as an rst link so it is clickable in the docs

Reply:
right, I forgot that sphinx will translate the docstring as well. I will make the correction right away.